backend/chain.go

```diff
@@ -368,6 +368,84 @@ func (g *gnoZenaoChain) UserAddress(userID string) string {
 	return gnolang.DerivePkgAddr(g.userRealmPkgPath(userID)).String()
 }
 
+// CreatePoll implements ZenaoChain
+func (g *gnoZenaoChain) CreatePoll(userID string, req *zenaov1.CreatePollRequest) error {
+	userRealmPkgPath := g.userRealmPkgPath(userID)
+	eventPkgPath := g.eventRealmPkgPath(req.EventId)
+	feedID := gnolang.DerivePkgAddr(eventPkgPath).String() + ":main"
+	options := ""
+	for _, option := range req.Options {
+		options += fmt.Sprintf(`%q, `, option)
+	}
+	kind := ""
+	if req.MultipleChoices {
+		kind = "pollsv1.POLL_KIND_MULTIPLE_CHOICE"
+	} else {
+		kind = "pollsv1.POLL_KIND_SINGLE_CHOICE"
+	}
+
+	broadcastRes, err := checkBroadcastErr(g.client.Run(gnoclient.BaseTxCfg{
+		GasFee:    "1000000ugnot",
+		GasWanted: 100000000,
+	}, vm.MsgRun{
+		Caller: g.signerInfo.GetAddress(),
+		Package: &gnovm.MemPackage{
```

Comment:
Why create a new realm?

This should be a `maketx run` since polls is a factory realm IMO

Reply:
it is a maketx run

Reply:
woops I got confused sorry